src/Kros.AspNetCore/Extensions/EndpointRouteBuilderExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Http.Connections;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Kros.AspNetCore.Extensions
+{
+    /// <summary>
+    /// Extensions for <see cref="IEndpointRouteBuilder"/>.
+    /// </summary>
+    public static class EndpointRouteBuilderExtensions
+    {
+        /// <summary>
+        /// Maps signalr hub with options from configuration.
+        /// </summary>
+        /// <param name="endpoints"><see cref="IEndpointRouteBuilder"/> where middleware is added.</param>
+        /// <param name="configuration">Configuration from which the options are loaded.
+        /// Configuration must contains ClientCredentialsAuthorization section.</param>
+        /// <param name="pattern">Signalr hub endpoint route pattern.</param>
+        /// <exception cref="InvalidOperationException">
+        /// When `HttpConnectionDispatcher` section is missing in configuration.
+        /// </exception>
+        public static void MapSignalRHubWithOptions<THub>(
+            this IEndpointRouteBuilder endpoints,
+            IConfiguration configuration,
+            [StringSyntax("Route")] string pattern)  where THub : Hub
+        {
+            HttpConnectionDispatcherOptions options = configuration.GetSection<HttpConnectionDispatcherOptions>();
```

Comment:
počuj, môžeš to prosím spraviť tak ako to robíme vo Frameworku (tak ako to robí MS)
 
![image](https://github.com/Kros-sk/Kros.AspNetCore/assets/5930822/21c0e4bb-6408-4e09-b646-29ab9e2c992f)

Má to výhodu, že vývojár si potom môže vybrať či to konfiguruje cez .json alebo priamo v kóde.

Reply:
dohodli sme sa ze to nebudeme robit, nakolko `mapHub` neberie tie options zo settings,  a ak by programator chcel options menit v kode, vzdy moze zavolat povodnu metodu z overloadom na nastavenie options